docs/admin/localhost.md

```diff
@@ -0,0 +1,74 @@
+# Localhot Installation
+
+To try out the software, you can install it on Ubuntu Server 22.04
+Operating System. The setup requires a
+machine which can spare 16GB RAM, 8 vCPUs and 50GB Hard Disk
+space to the vagrant box.
+A successful installation will create a setup
+similar to the one shown in the figure.
+
+![Single host install](./single-host.png)
+
+A one-step installation script is provided on this page. This script sets up
+the DTaaS software with a user.
+You can use it to check a test installation of DTaaS software.
+
+## Pre-requisites
+
+### 1. Gitlab OAuth application
+
+The DTaaS react website requires Gitlab OAuth provider.
+If you need more help with this step, please see
+the [Authentication page](client/auth.md).
```

Comment:
It is sufficient to have [user-owned oauth application](https://docs.gitlab.com/ee/integration/oauth_provider.html#create-a-user-owned-application) for localhost and trial installations. Pæease mention this point in information box both here and on trial installation page.

Please also create a section at the 3nd of _auth.md_  mentioning about the use of user-owned oauth application.

Reply:
@prasadtalasila  I don't understand the second part about the auth.md for the use of user-owned oauth

Also what the 3nd section of auth.md is